Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.3
+RUN apt-get update
+RUN apt-get install -y curl build-essential
+RUN go get github.com/tools/godep
+
+# Gearcmd
+RUN curl -L https://github.com/Clever/gearcmd/releases/download/v0.3.1/gearcmd-v0.3.1-linux-amd64.tar.gz | tar xz -C /usr/local/bin --strip-components 1
+
+# copy source code
+RUN mkdir -p /go/src/github.com/Clever/gearman-load-logger
+ADD . /go/src/github.com/Clever/gearman-load-logger
+
+# set workdir to find saved godeps
+WORKDIR /go/src/github.com/Clever/gearman-load-logger
+
+# build source code using godep
+RUN rm -rf /go/src/github.com/Clever/gearman-load-logger/Godeps/_workspace/pkg/
```

Comment:
I should probably add a comment on my [dockerfile](https://github.com/Clever/stoked/blob/master/Dockerfile) to give a better explanation of this line - it's only needed if the `Godeps/_workspace/pkg` folder was already created by a previous godep process.  

In my case, Drone first run tests, by first doing `make`, `make tests` ([makefile here](https://github.com/Clever/stoked/blob/master/Makefile)) which uses godep to install dependencies.  So when it comes time to create the Docker image, the pkg folder is already created by the tests and interferes.  

Since you're not using godep in your tests you probably don't need this... but since you're now using godep, it might make sense to update your makefile as well :).